bom/application/pom.xml

```diff
@@ -291,6 +281,17 @@
                 <type>pom</type>
             </dependency>
 
+            <!-- RESTEasy dependencies, imported as a BOM -->
+            <!-- As RESTEasy also imports Jackson, we need to import its BOM after the Jackson BOM -->
+            <!-- As RESTEasy also imports Vert.x transitively, we need to import its BOM after the Vert.x BOM -->
+            <dependency>
+                <groupId>org.jboss.resteasy</groupId>
+                <artifactId>resteasy-bom</artifactId>
+                <version>${resteasy.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
Why did you move the BOM? The BOM order is important so I wouldn't move it except if we have a good reason.

Reply:
There was a conflict with the version of Vert.x the RESTEasy BOM was importing. It was causing compile errors. I just moved it below the Vert.x import.

Reply:
Why is the RESTEasy BOM including a version of Vert.x? This doesn't seem like a normal behavior.

Reply:
RESTEasy has a Vert.x module (that is different to the Vert.x support we provide).

Reply:
Yes sure but it shouldn't include a version for Vert.x artifacts.

I put the BOM back where it was and I don't have any compilation issue.

I'll push to a branch in my fork to see what CI has to say.

Reply:
Also I made a diff of the effective POM and there's no difference except for the position. BOMs are a sensible matter so I prefer not changing it if we can avoid it.